website/docs/reference/resource-configs/grants.md

```diff
@@ -98,152 +72,117 @@ The `meta` config can also be defined under the `seeds` config block in `dbt_pro
 <File name='snapshots/schema.yml'>
 
 ```yml
-version: 2
-
 snapshots:
   - name: snapshot_name
-    config:
-      meta: {<dictionary>}
-
-    columns:
-      - name: column_name
-        meta: {<dictionary>}
-
+    config:  
+      grants:
+        select: ['reporter', 'bi']
 ```
 
 </File>
 
-The `meta` config can also be defined:
+The `grants` config can also be defined:
+
 - under the `snapshots` config block in `dbt_project.yml`
 - in a `config()` Jinja macro within a snapshot's SQL block
 
 See [configs and properties](configs-and-properties) for details.
 
 </TabItem>
+</Tabs>
 
-<TabItem value="analyses">
-
-The `meta` config is not currently supported for analyses.
-
-</TabItem>
+## Definition
 
-<TabItem value="macros">
+You can use the `grants` field to set permissions or grants for a resource. These grants will be compiled into the `manifest.json` file complied by dbt.
 
-<File name='macros/schema.yml'>
+## Database-specific requirements and notes
 
-```yml
-version: 2
+While we try to standardize the terms we use to describe different features, you will always find nuances in different databases. This section outlines some of those database-specific requirements and notes.
 
-macros:
-  - name: macro_name
-    meta: {<dictionary>}
+### Common syntax 
 
-    arguments:
-      - name: argument_name
-        meta: {<dictionary>}
+In our examples, you find terms like `select` and `another_user` because many databases use these terms, but be aware of the syntax your own database supports:
 
-```
+* Privileges: A right to perform an action in a database.
+* Grantees: A way to manage privileges. Recipients of granted privileges, also called "principals." Grantees can be a user, a group of users, a role held by users (Snowflake), a service account (GCP), and more.
 
-</File>
+<WHCode>
 
-</TabItem>
+<div warehouse="BigQuery">
 
-<TabItem value="exposures">
+- Be aware that the `grants` config is unrelated to the `grant_access_to` config:
+  - **`grants_access_to`:** Enables you to set up authorized views. When configured, dbt provides authorized view access to other datasets, without leaking additional data from those datasets. Fore more on this, see [BigQuery configurations: Authorized views](/reference/resource-configs/bigquery-configs#authorized-views)
+  - **`grants`:** Provides specific permissions to users, groups, or service accounts for managing access to datasets you're producing with dbt.You could grant a user, group or service account access to an authorized view set up by the `grants_access_to` feature.
+- Use BigQuery-specific grantee and privilege names. 
+  * Use `user:jeremy@dbtlabs.com` (do not use `jerco_user`)
+  * Use  `roles/bigquery.dataViewer` (do not use `select`)
```

Comment:
This `<WHCode>` mechanism is really cool!

A good bet, here and for all database-specific idiosyncrasies, is to link out to their docs:
```suggestion
- Use BigQuery-specific grantee and privilege names. 
  * Use `user:jeremy@dbtlabs.com` (do not use `jerco_user`)
  * Use  `roles/bigquery.dataViewer` (do not use `select`)
```
- Understanding GCP "roles" (what we call "privileges"): https://cloud.google.com/iam/docs/understanding-roles
- How to format grantees: https://cloud.google.com/bigquery/docs/reference/standard-sql/data-control-language#user_list